src/api/lib/tasks/dev/requests.rake

```diff
@@ -199,5 +199,357 @@ namespace :dev do
 
       puts "* Request with delete action #{request.number} has been created."
     end
+
+    desc 'Copy last 10 requests from openSUSE:Factory'
+    task requests_from_open_suse_factory: :development_environment do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      users = []
+      admin = User.get_default_admin
+      User.session = admin
+
+      # Setup interconnect
+      remote_proj = Project.find_by_name('openSUSE.org')
+      if remote_proj.nil?
+        print_message 'Creating the remote project for interconnect'
+
+        remote_proj = RemoteProject.new(name: 'openSUSE.org',
+                                        remoteurl: 'https://api.opensuse.org/public',
+                                        title: 'Standard OBS instance at build.opensuse.org',
+                                        description: 'This instance delivers the default build targets for OBS.')
+        remote_proj.store
+        FetchRemoteDistributionsJob.perform_now
+      end
+
+      clone_project(project_name: 'openSUSE:Factory', users: users)
+
+      # Get the list of requests from openSUSE:Factory
+      print_message 'Getting the list of requests'
+      url = "#{base_api_url}/request"
+      params = { view: 'collection', project: 'openSUSE:Factory', limit: '10', states: 'new,review' }
```

Comment:
Not sure if the "first 10" is the best decision. They are probably not very collaborated on.

Reply:
This requires modification. Currently, it only retrieves the oldest 10 requests in `review` state, which are dormant with minimal activity. I will explore methods to fetch the latest requests instead of the oldest ones.

Reply:
10 from the middle sounds good to me.